optihood/Pareto front comparison.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
This file should be in the personal repository, not in optihood. @vjacquot could you remove this file from here and add it to the personal repo?

Reply:
noted

Reply:
@vjacquot / @xavijoba could you take care of this before I merge the pull request?

Reply:
Hi @nehadimri1991 
This was handle by @vjacquot last week ! 
Can you know finish the merge ?